From a401425e791d2e1d281f833af06fe9c4eee86bed Mon Sep 17 00:00:00 2001
From: Vladimir Ermakov <vooon341@gmail.com>
Date: Fri, 16 Aug 2013 06:43:07 +0400
Subject: [PATCH 03/15] ARM: dts: imx6qdl-wandboard: add bluetooth control
 lines for bcm4329

This patch sets the required pins to enable the bluetooth module.

Signed-off-by: Vladimir Ermakov <vooon341@gmail.com>
---
 arch/arm/boot/dts/imx6qdl-wandboard.dtsi | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/arch/arm/boot/dts/imx6qdl-wandboard.dtsi b/arch/arm/boot/dts/imx6qdl-wandboard.dtsi
index 291c527..a7796ce 100644
--- a/arch/arm/boot/dts/imx6qdl-wandboard.dtsi
+++ b/arch/arm/boot/dts/imx6qdl-wandboard.dtsi
@@ -100,6 +100,9 @@
 				MX6QDL_PAD_ENET_TXD1__GPIO1_IO29 0x80000000 /* WL_HOST_WAKE */
 				MX6QDL_PAD_ENET_TXD0__GPIO1_IO30 0x80000000 /* WL_WAKE */
 				MX6QDL_PAD_EIM_D29__GPIO3_IO29   0x80000000
+				MX6QDL_PAD_EIM_DA13__GPIO3_IO13  0x80000000 /* BT_ON */
+				MX6QDL_PAD_EIM_DA14__GPIO3_IO14  0x80000000 /* BT_WAKE */
+				MX6QDL_PAD_EIM_DA15__GPIO3_IO15  0x80000000 /* BT_HOST_WAKE */
 			>;
 		};
 
-- 
1.9.1

